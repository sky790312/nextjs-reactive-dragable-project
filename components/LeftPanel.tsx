import React from 'react'
import styled from 'styled-components'
import Pages from '@/components/Pages'
import Elements from '@/components/Elements'

const LeftPanel: React.FC = () => {
  return (
    <LeftPanelWrapper>
      <Pages />
      <Elements />
    </LeftPanelWrapper>
  )
}

export default LeftPanel

const LeftPanelWrapper = styled.div`
  padding: 8px;
`
