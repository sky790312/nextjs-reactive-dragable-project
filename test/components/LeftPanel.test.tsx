import React from 'react'
import { render, withProviders } from '@/test/testUtils'
import LeftPanel from '@/components/LeftPanel'

describe('LeftPanel component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(withProviders(<LeftPanel />), {})
    expect(asFragment()).toMatchSnapshot()
  })
})
