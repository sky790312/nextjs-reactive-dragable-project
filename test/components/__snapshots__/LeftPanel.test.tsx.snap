// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftPanel component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="LeftPanel__LeftPanelWrapper-q1m529-0 klFdKt"
  >
    <div
      class="Pages__PagesWrapper-hol3qz-0 fQzYNz"
    >
      <h4>
        Pages
      </h4>
    </div>
    <div
      class="Elements__ElementsWrapper-vrh229-0"
    >
      <h4>
        Elements
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
